integrations/wallets/metamask.md

```diff
@@ -45,7 +45,160 @@ Moonbase Alpha TestNet:
 
 ## Step-by-step Tutorials
 
-In the case that you are interested in a more detailed step-by-step guide, you can go to our specific tutorials:
+In the case that you are interested in more detailed, step-by-step guides to configure MetaMask to Moonbeam, you can go to our specific tutorials:
 
  - MetaMask on a [Moonbeam development node](/getting-started/local-node/using-metamask/)
  - MetaMask on [Moonbase Alpha](/getting-started/testnet/metamask/)
+
+## Integrate MetaMask into a DApp
+
+With the release of MetaMask's [Custom Networks API](https://consensys.net/blog/metamask/connect-users-to-layer-2-networks-with-the-metamask-custom-networks-api/?utm_content=156337458&utm_medium=social&utm_source=twitter&hss_channel=tw-3278906401), users can be prompted to add Moonbeam's Testnet, Moonbase Alpha. This section will take you through the process of adding a "Connect to Moonbase Alpha" button that will prompt users to connect their MetaMask account(s) to Moonbase Alpha. Your users will no longer need to know or worry about Moonbase Alpha's network configurations and adding a custom network to MetaMask. To interact with Moonbeam from your dApp, all users will need to do is click a few buttons to connect to Moonbase Alpha and get started.
```

Comment:
In this URL you can actually remove everything after `networks-api/?` (After the /) This is just a cross reference to where you found the article.

The URL would be: https://consensys.net/blog/metamask/connect-users-to-layer-2-networks-with-the-metamask-custom-networks-api/